superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -305,6 +399,7 @@ const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
       case MENU_KEYS.EXPLORE_CHART:
         // eslint-disable-next-line no-unused-expressions
         props.logExploreChart?.(props.slice.slice_id);
+        window.open(props.exploreUrl, '_blank');
```

Comment:
Is popping open in  `_blank` the right default here, or should it open in the same window?

Reply:
I took from the same behavior as the click, which is a new window. But we could change it if we need to?